haiku-libs/sagebrush/sagebrush-1.0.0.recipe

```diff
@@ -0,0 +1,54 @@
+SUMMARY="An interface library for image editor"
+DESCRIPTION="SageBrush is an interface library for image editor. It provides \
+a kit to build an image editor easily without starting from scratch."
+HOMEPAGE="https://github.com/HaikuArchives/SageBrush"
+COPYRIGHT="2012 Vladislav Burundukov"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="d9b3945cb78c7070aa8b48d2e0bc6a2007fa3601"
+SOURCE_URI="https://github.com/HaikuArchives/SageBrush/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="d152e0d572a8eb0b8fc39a531cc1466827a9b96526bc5669dd181ef77ed32940"
+SOURCE_FILENAME="SageBrush-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="SageBrush-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 !x86_64"
+
+PROVIDES="
+	sagebrush$secondaryArchSuffix = $portVersion
+	lib:libsagebrush$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	sagebrush${secondaryArchSuffix}_devel
+	devel:libsagebrush$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	sagebrush$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	"
+
+BUILD()
+{
+	gcc -fPIC -shared -o libsagebrush.so -Isupport -Iinterface \
+		support/*.cpp interface/*.cpp
+}
+
+INSTALL()
+{
+	mkdir -p $libDir
+	cp -a libsagebrush.so $libDir
+	mkdir -p $includeDir
+	cp support/*.h $includeDir
```

Comment:
I'mnot sure if it is a good idea to put all the header files to the top level in the $includeDir. Maybe you can create a sagebrush folder for them. Is anything depends on it?

Reply:
I think it shouldn't be a dependence. Putting header files like this makes test projects build successfully without specifing an include path or changing the codes manually.